### PR TITLE
Hit the API

### PR DESCRIPTION
**Pull request by alef-garrido**
- Create a new game using the API.
- Implement the "Refresh" button (receiving data from the API and parsing the JSON).
- Implement the form "Submit" button (sending data to the API).

### CODE REVIEW

File: src/utils/fetchData.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      user,
+      score,
+    }),
+    headers: {
+      'Content-Type': 'application/json; charset=UTF-8',
+    },
+  })
+    .then((res) => res.json())
+    .then((json) => json)
+}
```
Comment (george-shammar) [inline src/utils/fetchData.js:30]: - [ ]  Please, let's use `async` and `await` with the `fetch` method to send and receive data.
- [ ] Please, include only relevant information in your README. You have provided instructions on how to run tests that you have not implemented in your application. Please, let's fix this.

File: src/utils/form.js [hunk 2]
```diff
@@ -2,10 +2,10 @@ const container = document.getElementById('addScore');
...
 const addForm = `
 <h2>Add your Score</h2>
-  <form action="#">
-    <input type="text" placeholder="Your name">
-    <input type="text" placeholder="Your score">
-    <input type="button" value="Submit">
+  <form id="addScore">
+    <input type="text" name="user" placeholder="Your name">
+    <input type="text" name="score" placeholder="Your score">
+    <input  class="addScore" type="button" value="submit">
```
Comment (george-shammar) [inline src/utils/form.js:8]: [OPTIONAL]

- [ ] Name and score fields should be properly validated to ensure user cannot submit blank entries. Please consider this.

**Review by george-shammar — CHANGES_REQUESTED** [review]
Hi @alef-garrido , 

Good job so far!

:heavy_check_mark: Your App runs as expected. I could submit user data and it reflects on the score board. :+1: 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/utils/fetchData.js (hunk 1 above)
Comment (alef-garrido) [reply src/utils/fetchData.js:30]: Hi @george-shammar, 
Thank you for your time and thoughtful comments, i just want to point out the fact that  async  and await are used when the fetch actions are actually called inside the module consumeApi.js. Could this be enough to consider this point as resolved?

File: src/utils/consumeApi.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    }
+  }
+
+  static myRequestGet = async () => {
+    try {
+      const getData = await apiActions.getData(`${API}games/${this.reqId}/scores/`);
+      this.reqScore = getData;
+    } catch (err) {
+      console.error(err);
+    }
```
Comment (alef-garrido) [inline src/utils/consumeApi.js:36]: @george-shammar  Please consider the highlighted code

**Review by Georjane — APPROVED** [review]
## Status: PR Approved 💯 🥇 
#### Hello @alef-garrido , 

Nice game you have there and great job making the requested changes. Your project is complete!  You have met all requirements for this project. It is a thumbs up for me. 👍🏽 You can go ahead and merge this PR. 
Congratulations! 🎉 👏👏

_Good luck with your next project and Happy coding!_ 💻

**Pull request merged**